### DISCUSSION TITLE
Any guide for playfab GDSK?

### DISCUSSION THREAD

**Discussion by RProduction** [Structed/godot-playfab#142] [Q&A]
Hello, just found this plugin. But how about to implement server build? Or this plugin is used for general playfab API only? Any guide about how to host or deploy godot server into playfab?

After looking at playfab API, seem there are many feature available. Looking at the plugin code, seem the plugin only cover part of playfab feature. Can I combine playfab restapi with this plugin? Like matchmaking? To automatically spin up new server.

For multiplayer server hosting, seem godot-server-docker can work.

**Comment by Structed** [comment]
godot-playfab is purely for the REST Web API.

I do want to cover more, but my time doesn't allow for more right now! If you would like to contribute GDSK integration, that would be absolutely fantastic!
